opendbc/car/hyundai/values.py

```diff
@@ -126,6 +126,8 @@ class HyundaiFlags(IntFlag):
 
   ALT_LIMITS_2 = 2 ** 26
 
+  HAS_CLUSTER_LANELINES = 2 ** 27
```

Comment:
is this something we can detect from the stock camera messages instead of hardcoding?

Reply:
There isn't much of a pattern on the camera FWs to distinguish them, as some camera FWs are shared across different models that don't share the same configs.

Reply:
I also checked to do this last year and wasn't able to detect it easily